--- recumbent-bike-fast-rv.brf	2025-12-28 17:54:52.647267902 +0100
+++ allroad-rv.brf	2025-12-28 18:09:55.994329440 +0100
@@ -9,7 +9,7 @@
 
 assign dismount_cost = 1000  # %dismount_cost% | Acceptable detour in meters to avoid dismount, from 1 000 000 no dismount at all | number
 assign avoid_cycleways = 5   # %avoid_cycleways% | Avoid cycleways (0 - 10): 0 = don't avoid, 10 = strongly avoid | number
-assign avoid_elevation = 10  # %avoid_elevation% | Avoid elevation (0 - 10): 0 = don't avoid, 10 = strongly avoid | number
+assign avoid_elevation = 2   # %avoid_elevation% | Avoid elevation (0 - 10): 0 = don't avoid, 10 = strongly avoid | number
 
 assign totalMass  = 90    # %totalMass% | Total weight of the vehicle including rider and luggage in kg, for estimation of travel time | number
 assign maxSpeed   = 90    # Maximum speed of the vehicle in km/h
@@ -65,7 +65,7 @@
 
 assign turncost 
 	switch junction=roundabout 100
-	400  # Cost for a 90 degree turn. Other angles are calculated with turncost * cos(angle). This does not apply to roundabouts.
+	200  # Cost for a 90 degree turn. Other angles are calculated with turncost * cos(angle). This does not apply to roundabouts.
 
 assign initialclassifier
 	switch route=ferry 2
@@ -244,12 +244,12 @@
 
 assign no_river_penalty
 	switch consider_river
-		switch estimated_river_class=   2
-		switch estimated_river_class=1  1.5
-		switch estimated_river_class=2  1.1
-		switch estimated_river_class=3  0.7
-		switch estimated_river_class=4  0.3
-		switch estimated_river_class=5  0.15
+		switch estimated_river_class=   1
+		switch estimated_river_class=1  0.5
+		switch estimated_river_class=2  0.4
+		switch estimated_river_class=3  0.25
+		switch estimated_river_class=4  0.15
+		switch estimated_river_class=5  0.1
 		switch estimated_river_class=6  0
 		99
 	0
@@ -269,78 +269,78 @@
 
 assign surfacepenalty
 	add 
-		switch embedded_rails=tram|yes|rail 5
+		switch embedded_rails=tram|yes|rail 3
 		0
 		
 		switch surface=asphalt|paved 0
-		switch concrete=plates 0.5
-		switch concrete=lanes 1
-		switch surface=concrete|paving_stones 0.2
-		switch surface=wood|metal 1
-		switch surface=sett|grass_paver 15
-		switch surface=cobblestone 50  # add |unhewn_cobblestone when it's added to the routing data
-		switch surface=fine_gravel|compacted 10
-		switch surface=gravel|pebblestone 30
-		switch surface=ground|grass|unpaved|dirt|earth|clay 25
-		switch surface=mud|sand 200
+		switch concrete=plates 0.1
+		switch concrete=lanes 0.1
+		switch surface=concrete|paving_stones 0
+		switch surface=wood|metal 0.2
+		switch surface=sett|grass_paver 1
+		switch surface=cobblestone 5  # add |unhewn_cobblestone when it's added to the routing data
+		switch surface=fine_gravel|compacted 1
+		switch surface=gravel|pebblestone 10
+		switch surface=ground|grass|unpaved|dirt|earth|clay 10
+		switch surface=mud|sand 100
 		switch tracktype=grade1	0
-		switch tracktype=grade2	8
-		switch tracktype=grade3	15
-		switch tracktype=grade4	20
-		switch tracktype=grade5	25
+		switch tracktype=grade2	4
+		switch tracktype=grade3	7
+		switch tracktype=grade4	9
+		switch tracktype=grade5	10
 		0
 
 assign uphillsurfacepenalty
 	add 
-		switch embedded_rails=tram|yes|rail 2
+		switch embedded_rails=tram|yes|rail 1
 		0
 		
 		switch surface=asphalt|paved 0
 		switch concrete=plates 0
-		switch concrete=lanes 0.5
+		switch concrete=lanes 0
 		switch surface=concrete|paving_stones 0
-		switch surface=wood|metal 0.5
-		switch surface=sett|grass_paver 3
-		switch surface=cobblestone 10  # add |unhewn_cobblestone when it's added to the routing data
-		switch surface=fine_gravel|compacted 5
-		switch surface=gravel|pebblestone 10
-		switch surface=ground|grass|unpaved|dirt|earth|clay 20
-		switch surface=mud|sand 100
+		switch surface=wood|metal 0
+		switch surface=sett|grass_paver 0.2
+		switch surface=cobblestone 3  # add |unhewn_cobblestone when it's added to the routing data
+		switch surface=fine_gravel|compacted 0.5
+		switch surface=gravel|pebblestone 5
+		switch surface=ground|grass|unpaved|dirt|earth|clay 5
+		switch surface=mud|sand 50
 		switch tracktype=grade1	0
-		switch tracktype=grade2	4
-		switch tracktype=grade3	10
-		switch tracktype=grade4	15
-		switch tracktype=grade5	20
+		switch tracktype=grade2	3
+		switch tracktype=grade3	4
+		switch tracktype=grade4	5
+		switch tracktype=grade5	5
 		0
 
 assign downhillsurfacepenalty
 	add 
-		switch embedded_rails=tram|yes|rail 20
+		switch embedded_rails=tram|yes|rail 10
 		0
 		
 		switch surface=asphalt|paved 0
-		switch concrete=plates 3
-		switch concrete=lanes 10
-		switch surface=concrete|paving_stones 1
-		switch surface=wood|metal 20
-		switch surface=sett|grass_paver 30
-		switch surface=cobblestone 100  # add |unhewn_cobblestone when it's added to the routing data
-		switch surface=fine_gravel|compacted 50
-		switch surface=gravel|pebblestone 100
-		switch surface=ground|grass|unpaved|dirt|earth|clay 100
-		switch surface=mud|sand 1000
+		switch concrete=plates 0.5
+		switch concrete=lanes 1
+		switch surface=concrete|paving_stones 0
+		switch surface=wood|metal 5
+		switch surface=sett|grass_paver 2
+		switch surface=cobblestone 20  # add |unhewn_cobblestone when it's added to the routing data
+		switch surface=fine_gravel|compacted 10
+		switch surface=gravel|pebblestone 50
+		switch surface=ground|grass|unpaved|dirt|earth|clay 40
+		switch surface=mud|sand 500
 		switch tracktype=grade1	0
-		switch tracktype=grade2	40
-		switch tracktype=grade3	50
-		switch tracktype=grade4	70
-		switch tracktype=grade5	100
+		switch tracktype=grade2	25
+		switch tracktype=grade3	30
+		switch tracktype=grade4	35
+		switch tracktype=grade5	40
 		0
 
 assign smoothnesspenalty
 	switch smoothness=excellent|very_good 0
-	switch smoothness=good 0.05
-	switch smoothness=intermediate 2
-	switch smoothness=bad 10
+	switch smoothness=good 0
+	switch smoothness=intermediate 1.5
+	switch smoothness=bad 5
 	switch smoothness=very_bad 40
 	switch smoothness=horrible|very_horrible|impassable 100
 	0
@@ -348,25 +348,25 @@
 assign uphillsmoothnesspenalty
 	switch smoothness=excellent|very_good 0
 	switch smoothness=good 0
-	switch smoothness=intermediate 0.3
-	switch smoothness=bad 3
+	switch smoothness=intermediate 0.2
+	switch smoothness=bad 2
 	switch smoothness=very_bad 10
 	switch smoothness=horrible|very_horrible|impassable 100
 	0
 
 assign downhillsmoothnesspenalty
 	switch smoothness=excellent|very_good 0
-	switch smoothness=good 0.1
-	switch smoothness=intermediate 5
-	switch smoothness=bad 30
+	switch smoothness=good 0
+	switch smoothness=intermediate 3
+	switch smoothness=bad 20
 	switch smoothness=very_bad 100
 	switch smoothness=horrible|very_horrible|impassable 1000
 	0
 
 assign mtb_scale_cost
 	switch mtb:scale=0	0
-	switch mtb:scale=0+	20
-	switch mtb:scale=1	50  # can be steep
+	switch mtb:scale=0+	10
+	switch mtb:scale=1	20  # can be steep
 	switch not mtb:scale=	9999  # all other mtb:scale
 	0
 
@@ -406,21 +406,21 @@
 	max 1
 	switch highway=motorway|motorway_link 10000
 	switch route=ferry 10.67
-	switch highway=trunk|trunk_link 4
-	switch highway=primary|primary_link 3
-	switch highway=secondary|secondary_link 1.4
+	switch highway=trunk|trunk_link 8
+	switch highway=primary|primary_link 5
+	switch highway=secondary|secondary_link 2
 	switch highway=tertiary|tertiary_link 1.1
 	switch highway=unclassified 1
-	switch highway=residential 1.2
+	switch highway=residential 1.1
 	switch highway=service 1
 	switch highway=track 1
 	switch cycleway 1
-	switch highway=living_street 2
+	switch highway=living_street 1.5
 	switch highway=road 3
-	switch footway 40
-	switch highway=steps 100
-	switch highway=path 1.5
-	switch highway=bridleway 2
+	switch footway 20
+	switch highway=steps 50
+	switch highway=path 1.1
+	switch highway=bridleway 1.2
 	20
 
 
@@ -460,10 +460,10 @@
 	switch cycleway 1
 	switch highway=living_street 1
 	switch highway=road 2
-	switch footway 5
-	switch highway=steps 40
+	switch footway 2
+	switch highway=steps 30
 	switch highway=path 1
-	switch highway=bridleway 1.5
+	switch highway=bridleway 1
 	20
 
 
@@ -492,21 +492,21 @@
 	max 1
 	switch highway=motorway|motorway_link 10000
 	switch route=ferry 10.67
-	switch highway=trunk|trunk_link 1.5
-	switch highway=primary|primary_link 1.2
+	switch highway=trunk|trunk_link 3
+	switch highway=primary|primary_link 2
 	switch highway=secondary|secondary_link 1
 	switch highway=tertiary|tertiary_link 1
 	switch highway=unclassified 1
-	switch highway=residential 3
+	switch highway=residential 1.5
 	switch highway=service 1
-	switch highway=track 1.5
-	switch cycleway 2
-	switch highway=living_street 6
+	switch highway=track 1
+	switch cycleway 1.1
+	switch highway=living_street 5
 	switch highway=road 6
-	switch footway 50
+	switch footway 40
 	switch highway=steps 300
 	switch highway=path 10
-	switch highway=bridleway 3
+	switch highway=bridleway 2
 	20
 
 
