--- recumbent-bike-fast-rv.brf	2025-12-28 13:22:56.573503839 +0100
+++ allroad-rv.brf	2025-12-28 13:22:53.484433227 +0100
@@ -9,7 +9,7 @@
 
 assign dismount_cost = 1000  # %dismount_cost% | Acceptable detour in meters to avoid dismount, from 1 000 000 no dismount at all | number
 assign avoid_cycleways = 5   # %avoid_cycleways% | Avoid cycleways (0 - 10): 0 = don't avoid, 10 = strongly avoid | number
-assign avoid_elevation = 10  # %avoid_elevation% | Avoid elevation (0 - 10): 0 = don't avoid, 10 = strongly avoid | number
+assign avoid_elevation = 2  # %avoid_elevation% | Avoid elevation (0 - 10): 0 = don't avoid, 10 = strongly avoid | number
 
 assign totalMass  = 90    # %totalMass% | Total weight of the vehicle including rider and luggage in kg, for estimation of travel time | number
 assign maxSpeed   = 90    # Maximum speed of the vehicle in km/h
@@ -65,7 +65,7 @@
 
 assign turncost 
 	switch junction=roundabout 100
-	400  # Cost for a 90 degree turn. Other angles are calculated with turncost * cos(angle). This does not apply to roundabouts.
+	200  # Cost for a 90 degree turn. Other angles are calculated with turncost * cos(angle). This does not apply to roundabouts.
 
 assign initialclassifier
 	switch route=ferry 2
@@ -244,12 +244,12 @@
 
 assign no_river_penalty
 	switch consider_river
-		switch estimated_river_class=   2
-		switch estimated_river_class=1  1.5
-		switch estimated_river_class=2  1.1
-		switch estimated_river_class=3  0.7
-		switch estimated_river_class=4  0.3
-		switch estimated_river_class=5  0.15
+		switch estimated_river_class=   1
+		switch estimated_river_class=1  0.5
+		switch estimated_river_class=2  0.4
+		switch estimated_river_class=3  0.25
+		switch estimated_river_class=4  0.15
+		switch estimated_river_class=5  0.1
 		switch estimated_river_class=6  0
 		99
 	0
